pkg/neonvm/controllers/vm_controller.go

```diff
@@ -239,20 +235,15 @@ func (r *VMReconciler) doFinalizerOperationsForVirtualMachine(ctx context.Contex
 	log := log.FromContext(ctx)
 
 	// The following implementation will raise an event
-	r.Recorder.Event(vm, "Warning", "Deleting",
-		fmt.Sprintf("Custom Resource %s is being deleted from the namespace %s",
-			vm.Name,
-			vm.Namespace))
+	log.Info("Deleting VirtualMachine")
```

Comment:
Do we really need that logline? Realistically it conveys almost 0 info. I'd suggest adding either which machine gets deleted or just remove this altogether

Reply:
The VM object it refers to is already included in the log context IIRC